### PR TITLE
test oldest and newest Ruby on AppVeyor; enable pygments tests

### PR DESCRIPTION
**Pull request by mojavelinux**

### CODE REVIEW

**Comment by mojavelinux** [comment]
Well that partially explains the problem. The AppVeyor builds aren't actually running.

**Comment by slonopotamus** [comment]
Hehe

File: .appveyor.yml [hunk 1]
```diff
@@ -2,15 +2,17 @@ version: '{build}'
 skip_tags: true
 clone_depth: 2
 environment:
+  PYGMENTS_VERSION: '~> 1.2.0'
   matrix:
-  # see https://www.appveyor.com/docs/build-environment/#ruby for list of installed Ruby runtimes
+  # see https://www.appveyor.com/docs/linux-images-software/#ruby for list of installed Ruby runtimes
```
Comment (slonopotamus) [inline .appveyor.yml]: Err... Why Linux?

Comment (mojavelinux) [reply .appveyor.yml]: Because their documentation is quirky.

**Pull request merged**